go.mod

```diff
@@ -5,8 +5,6 @@ go 1.17
 require (
 	github.com/go-logr/logr v1.2.0
 	github.com/grafana-operator/grafana-operator/v4 v4.0.1
-	github.com/operator-framework/api v0.10.3
```

Comment:
I would expected to see, grafana-operator removed as well 🤔 . Do you know where it is being referred?